### ISSUE TITLE
Training custom model

### ISSUE THREAD

**Issue by perces** [HRNet/Lite-HRNet#79]
I want to train a table key point detection model to detect the four corners of the table. What should I do?

**Comment by sang981113** [comment]
yes you could do that. but do you have the datasets to train the model?
